addons/zeus/functions/fnc_moduleToggleFlashlight.sqf

```diff
@@ -36,7 +36,11 @@ if (_toggle) then {
             private _pointer = (_x weaponAccessories (currentWeapon _x)) select 1;
 
             if (!(_pointer isEqualTo "") && {isNull (configfile >> "CfgWeapons" >> _pointer >> "ItemInfo" >> "Pointer")}) then {
-                _x enableGunLights "forceOn";
+                if (local _x) then {
+                    _x enableGunLights "forceOn";
+                } else {
+                    [QGVAR(enableFlashlight), [_x, "forceOn"], _x] call CBA_fnc_targetEvent;
```

Comment:
Target event handles locality on its own, so you don't need duplicate code. Probably minor performance difference, shouldn't matter here at all.